packages/multichain-account-service/src/MultichainAccountService.test.ts

```diff
@@ -832,6 +851,27 @@ describe('MultichainAccountService', () => {
 
       expect(isInProgress).toBe(false);
     });
+
+    it('creates a multichain account wallet with MultichainAccountService:createMultichainAccountWallet', async () => {
+      const { messenger, mocks } = setup({ accounts: [], keyrings: [] });
+
+      mocks.KeyringController.getKeyringsByType.mockImplementationOnce(
+        () => [],
+      );
+
+      mocks.KeyringController.addNewKeyring.mockImplementationOnce(() => ({
+        id: 'abc',
+        name: '',
+      }));
+
+      const wallet = await messenger.call(
+        'MultichainAccountService:createMultichainAccountWallet',
+        { mnemonic: MOCK_MNEMONIC },
+      );
+
+      expect(wallet).toBeDefined();
+      expect(wallet.entropySource).toBe('abc');
+    });
```

Comment:
I think this is a duplicate (we have it in the `describe('createMultichainAccountWallet', ...)` block):

```suggestion
```